barretenberg/cpp/src/barretenberg/goblin/goblin.hpp

```diff
@@ -39,7 +39,7 @@ class GoblinProver {
     using TranslationEvaluations = ECCVMProver::TranslationEvaluations;
     using TranslatorBuilder = bb::TranslatorCircuitBuilder;
     using TranslatorProver = bb::TranslatorProver;
-    using TranslatorProvingKey = bb::TranslatorFlavor::ProvingKey;
+    using TranslatorProvingKey = bb::TranslatorProvingKey;
```

Comment:
We are already in namespace `bb` so this line is doing nothing, and I'd even expect some compilers to complain about it (surprised GCC didn't).

Reply:
will remove